services/ui-src/src/styles/theme.ts

```diff
@@ -75,6 +75,7 @@ export const theme = extendTheme({
       error_darkest: "#981b1e",
       // neutrals
       white: "#ffffff",
+      custom_gray_lightest: "#f8f8f8",
       gray_lightest: "#f1f1f1",
```

Comment:
I'd probably just use the `gray_lightest` that's already here since it's only 7 points off on each value. Gotta keep the designers honest. 😉

Reply:
This is for highlighting a field that already has the `gray_lightest` background. We agreed to change the name to `gray_lightest_highlight`